src/eterna/mode/DesignBrowser/ViewSolutionOverlay.ts

```diff
@@ -237,10 +237,10 @@ export default class ViewSolutionOverlay extends ContainerObject {
                     puznid: this._props.puzzle.nodeID,
                     filter1: 'Id',
                     // TODO: Update website so that these can be camelcase
-                    /* eslint-disable @typescript-eslint/camelcase */
+                    /* eslint-disable @typescript-eslint/naming-convention */
                     filter1_arg1: this._props.solution.nodeID,
                     filter1_arg2: this._props.solution.nodeID
-                    /* eslint-enable @typescript-eslint/camelcase */
+                    /* eslint-enable camelcase */
```

Comment:
wait, shouldn't this match what you've disabled?

Reply:
That's correct - basically we already had overrides in specific areas of the code, which I kept, but the way that the rule was configured changed in a nontrivial way and I didn't want to spend too much time figuring that out when I have a significantly better lint setup in v2 anyways